app/models/hackbot/conversation.rb

```diff
@@ -12,6 +12,11 @@ def self.should_start?(_event, _team)
       raise NotImplementedError
     end
 
+    def self.mentions_name?(event, team)
+      event[:text].include?(team[:bot_username]) ||
+        event[:text].include?("<@#{team[:bot_user_id]}>")
```

Comment:
@paked: Thoughts on only responding to `@hackbot` mentions, not username mentions?